autoria.py

```diff
@@ -14,28 +14,34 @@
 class RiaAPI:
     """Python auto.ria.com API.
 
-    Python implementation of API intended for calulating
-    average used car prices that are sold on http://auto.ria.com
+    This class contains methods to work with auto.ria.com API:
+    get categories, bodystyles, states etc. lists with identifiers,
+    send a request to calculate average price
     """
 
     def __init__(self):
-        """Initialize a class."""
+        """Constructor."""
         self._base_url = 'http://api.auto.ria.com'
 
-    def _make_request(self, url: str) -> Union[dict, list]:
+    def _make_request(self, url: str, parameters: dict = None) \
```

Comment:
Didn't know about backslash is a bad practice :) Corrected using your first option.
And yep, I need line break here, because the line is 85 symbols long.